source/Target/Target.cpp

```
@@ -2538,6 +2538,14 @@ SwiftASTContextReader Target::GetScratchSwiftASTContext(
   return SwiftASTContextReader(GetSwiftScratchContextLock(), swift_ast_ctx);
 }
 
+SwiftASTContextReader Target::GetScratchSwiftASTContext(ValueObject &valobj,
+                                                        bool create_on_demand) {
+  Status error;
```

I don't like this error getting swallowed.